course_viewer/course_viewer/serializers.py

```diff
@@ -0,0 +1,53 @@
+from course_viewer.models import (
+    College,
+    Course,
+    MTUCourse,
+    State
+)
+from rest_framework import serializers
+
+
+class CollegeSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = College
+        fields = '__all__'
+
+
+class CourseSerializer(serializers.ModelSerializer):
+    course_mtu_course_id = serializers.CharField(
+                                source='course_mtu_course_id.mtu_course_id',
+                                read_only=True)
+    course_mtu_course_subject = serializers.CharField(
+                                source='course_mtu_course_id.mtu_subject',
+                                read_only=True)
+    course_college_code = serializers.CharField(
+                                source='course_college_code.college_code',
+                                read_only=True)
+    course_college_name = serializers.CharField(
+                                source='course_college_code.college_name',
+                                read_only=True)
+    course_state_name = serializers.CharField(
+                                source='course_state_code.state_name',
+                                read_only=True)
+    course_state_code = serializers.CharField(
+                                source='course_state_code.state_code',
+                                read_only=True)
```

Comment:
Basically all the variables in this could use some changing on the naming of them. Without the help text (such as when they get parsed to JSON output on a curl). Become rather unreadable and confusing without already knowing the output.

{
    "id": 1,
    "course_mtu_course_id": "2700",  // Okay it's MTU's course ID
    "course_mtu_course_subject": "HU", // Okay it's MTU's subject name
    "course_college_code": "10", // Course college code? for which course the MTU or transfer?
    "course_college_name": "FakeCollege", // Same as above
    "course_state_name": "Michigan", // Same as above
    "course_state_code": "MI", // Same as above
    "course_name": "Psychology 101", // From which? MTU or the transfering? also same as above
    "course_id": "PSY101", // same as above
    "course_credit": 3, // same as above
    "course_number": "101"// same as above, also now we're calling it a course number instead of ID
}